src/cmd/ksh93/sh/lex.c

```diff
@@ -258,7 +258,7 @@ Lex_t *sh_lexopen(Lex_t *lp, Shell_t *shp, int mode) {
 #ifdef DBUG
 extern int lextoken(Lex_t *);
 int sh_lex(Lex_t *lp) {
-    Shell_t *shp = lp->sh;
+    Shell_t *shell_interpreter = lp->sh;
```

Comment:
As much as I dislike a lot of the existing idioms (not to mention too short var names) this is a case where you should follow the existing idiom of naming this var `shp`. I want to be able to `grep shp` and find all the places this `Shell_t*` is used. I have no objections to establishing a different, clearer, idiom but that should be done in a separate change.

Reply:
👍 Got it, will make sure to isolate these kinds of changes in the future.